machine-config/digitalocean.vue

```diff
@@ -111,6 +112,16 @@ export default {
     'value.image': 'updateUsername',
   },
 
+  computed: {
+    isCreate() {
+      if (this.mode === _CREATE) {
+        return true;
+      }
+
+      return false;
```

Comment:
This could be simplified to return `this.mode === _CREATE`.

Reply:
I knew there had to be a cleaner way, thanks!